app/src/test/java/party_queue/myapplication/ExampleUnitTest.java

```diff
@@ -0,0 +1,15 @@
+package party_queue.myapplication;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * To work on unit tests, switch the Test Artifact in the Build Variants view.
+ */
+public class ExampleUnitTest {
```

Comment:
Is this file generated when one creates a project on android studio? If so, we probably don't want it in master